index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="first">
+        <div class="first-img"><img src="./images/Welcome.png" alt="welcome button at the top left"></div>
+        <div class="first-img" > <img src="./images/Union.png" alt="a union of three vectors"> </div>
```

Comment:
- [ ] Kindly wrap your hamburger button inside nav tag. This is for semantics reason, and the right position for convention and user experience for mobile devices.